### DISCUSSION TITLE
How to disable and stretch remaining LEDs for weird configuration

### DISCUSSION THREAD

**Discussion by zmatez** [awawa-dev/HyperHDR#818] [General]
Hello,
my desktop configuration looks like this:
![image](https://github.com/awawa-dev/HyperHDR/assets/43961846/9932ee53-8126-43e0-a231-6aec2724a987)
So I have 95 leds above monitor and 99 leds behind. Monitor is not at the center of the desk and so that strip above displays colors at wrong positions:
![image](https://github.com/awawa-dev/HyperHDR/assets/43961846/bb432f99-98c0-4768-8cdc-0c230bacdf44)
![image](https://github.com/awawa-dev/HyperHDR/assets/43961846/7c1de1b4-7782-4251-a988-98c5519b6d3e)
In order to achieve this configuration I created two instances one for strip one for screen, and joined them in JSON into one instance.

What I want to do is to disable some of the LEDs from the strip above and leave only those that are EXACTLY above screen, so colors would match what's displayed on the screen. The rest on the strip would be black. So lets say 10 from right and 35 from left side would be disabled - I can disable them manually, but the rest of them won't stretch to fill the blank area - so colors would still be off.
Any ideas how to workaround this?
![image](https://github.com/awawa-dev/HyperHDR/assets/43961846/3700f80f-ca63-4ce7-865d-fd7abdcdb5f3)

I run on WLED (by wifi), I have two separate instances for both.
Thanks in advance

**Comment by awawa-dev** [comment]
Hi
> **In order to achieve this configuration I created two instances one for strip one for screen, and joined them in JSON into one instance.**
What I want to do is to disable some of the LEDs from the strip above and leave only those that are EXACTLY above screen, so colors would match what's displayed on the screen. The rest on the strip would be black. So lets say 10 from right and 35 from left side would be disabled - I can disable them manually, but the rest of them won't stretch to fill the blank area - so colors would still be off.

For example: you can disable LEDs before joining the instances: for the temporary instance than reflects the top led strip create geometry with right/left (depending on if it's clockwise or counterclockwise connection) edge of 10 leds and left/right 35 leds and top 95 - 10 - 35 (top LEDs should have perfect alignment with the top screen edge) and then disable these 10 &  35 LEDs. Then join instances in the editor. These 10 & 35 LEDs wont be bother you anymore.
